lib/scrabble.rb

```diff
@@ -15,4 +27,27 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def prepare(word)
+    if word.class != Array
+      word.upcase!
+      word = word.chars
+    else
+      word
+    end
+  end
+
+  def return_score(word)
+    word.each do |letter|
+      points_array << point_values[letter].to_i
+    end
+    return points_array
```

Comment:
Don't think we need the explicit return here. `points_array` is the last line, and will be the value returned. Additionally we're not using the return of this method (line 13 doesn't store the result in anything).

Could potentially even use `word.map` on line 41, and then we wouldn't need to shovel into `points_array`, but would need to store the return value in a variable on line 13.